quipucords/scanner/satellite/six.py

```diff
@@ -389,6 +389,17 @@ def host_details(self, org_id, host_id, host_name):
         :returns: dictionary of host details
         """
         details = {}
+        sys_result = None
+        sys_results = self.inspect_result.systems.all()
```

Comment:
Why do systems.all() instead of doing a filter() here?

Reply:
done.